webapi/src/main/twirl/queries/sparql/v1/getResourceInfoGraphDB.scala.txt

```diff
@@ -0,0 +1,96 @@
+@*
+ * Copyright © 2015 Lukas Rosenthaler, Benjamin Geer, Ivan Subotic,
+ * Tobias Schweizer, André Kilchenmann, and André Fatton.
+ *
+ * This file is part of Knora.
+ *
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ *@
+
+@import org.knora.webapi.IRI
+
+@**
+ * Gets all the properties of a resource, along with the properties of its file values, but not
+ * including subproperties of knora-base:hasValue, using GraphDB.
+ *
+ * This template is used only by getResourceInfo.scala.txt.
+ *
+ * Since the triplestore type is GraphDB, we assume that inference is enabled, and we use it to optimise the generated
+ * SPARQL. Specifically, we use inference to return search results matching subproperties of Knora base properties
+ * such as knora-base:hasFileValue. This requires us to use GraphDB's GRAPH <http://www.ontotext.com/explicit>
+ * whenever we need to get explicit (non-inferred) statements.
+ *
+ * @param resourceIri the IRI of the resource to be queried.
+ *@
+@(resourceIri: IRI)
+
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix knora-base: <http://www.knora.org/ontology/knora-base#>
+
+SELECT ?prop ?obj ?isFileValue ?objPred ?objObj
+WHERE {
+    BIND(IRI("@resourceIri") AS ?resource)
+
+    ?resource rdf:type knora-base:Resource ;
+        knora-base:isDeleted false .
+
+    {
+        @* Query statements about the resource's file values. *@
+
+        ?resource knora-base:hasFileValue ?obj .
```

Comment:
this would behave differently than the Sparql in `getResourceInfoStandard.scala.txt` because there the plus operator is used.